docs/victoriametrics/relabeling.md

```diff
@@ -10,250 +10,198 @@ aliases:
-[Prometheus-compatible relabeling](https://docs.victoriametrics.com/vmagent/#relabeling)
-with [additional enhancements](https://docs.victoriametrics.com/vmagent/#relabeling-enhancements).
-
-The relabeling is mostly used for the following tasks:
-
-* Dropping unneeded scrape targets during [service discovery](https://docs.victoriametrics.com/sd_configs/#prometheus-service-discovery).
-  See [how to drop discovered targets](#how-to-drop-discovered-targets).
-* Adding or updating static labels at scrape targets. See [how to add labels to scrape targets](#how-to-add-labels-to-scrape-targets).
-* Copying target labels from another labels. See [how to copy labels in scrape targets](#how-to-copy-labels-in-scrape-targets).
-* Modifying scrape urls for discovered targets. See [how to modify scrape urls in targets](#how-to-modify-scrape-urls-in-targets).
-* Modifying `instance` and `job` labels. See [how to modify instance and job](#how-to-modify-instance-and-job).
-* Extracting label parts into another labels. See [how to extract label parts](#how-to-extract-label-parts).
-* Removing prefixes from target label names. See [how to remove prefixes from target label names](#how-to-remove-prefixes-from-target-label-names).
-* Removing some labels from discovered targets. See [how to remove labels from targets](#how-to-remove-labels-from-targets).
-* Dropping some metrics during scape. See [how to drop metrics during scrape](#how-to-drop-metrics-during-scrape).
-* Renaming scraped metrics. See [how to rename scraped metrics](#how-to-rename-scraped-metrics).
-* Adding labels to scraped metrics. See [how to add labels to scraped metrics](#how-to-add-labels-to-scraped-metrics).
-* Changing label values in scraped metrics. See [how to change label values in scraped metrics](#how-to-change-label-values-in-scraped-metrics).
-* Removing some labels from scraped metrics. See [how to remove labels from scraped metrics](#how-to-remove-labels-from-scraped-metrics).
-* Removing some labels from metrics matching some [series selector](https://prometheus.io/docs/prometheus/latest/querying/basics/#time-series-selectors).
-  See [how to remove labels from metrics subset](#how-to-remove-labels-from-metrics-subset).
 
-See also [relabeling docs at vmagent](https://docs.victoriametrics.com/vmagent/#relabeling).
+The relabeling cookbook provides practical examples and patterns for transforming your metrics data as it flows through VictoriaMetrics, helping you control what gets collected and how it's labeled.
+
+VictoriaMetrics and vmagent support Prometheus-style relabeling with [extra features](https://docs.victoriametrics.com/vmagent/#relabeling-enhancements) to enhance the functionality.
 
+## Relabeling Stages
 
-## How to remove labels from metrics subset
+Relabeling in Prometheus happens in two main stages: during service discovery (`relabel_configs`), and during scraping (`metric_relabel_configs`).
```

Comment:
```suggestion
Relabeling happens in two main stages: during service discovery (`relabel_configs`), and during scraping (`metric_relabel_configs`).
```

Reply:
It could also happen on remote write - see https://docs.victoriametrics.com/victoriametrics/vmagent/#relabeling